src/main/scala/prci/ClockBundles.scala

```diff
@@ -10,7 +10,7 @@ class ClockBundle(val params: ClockBundleParameters) extends Bundle
   val reset = Reset()
 }
 
-class ClockGroupBundle(val params: ClockGroupBundleParameters) extends Bundle
+class ClockGroupBundle(val params: ClockGroupBundleParameters, val names: Option[Seq[String]] = None) extends Bundle
```

Comment:
Would it be simpler to have the names be calculated by `ClockGroupBundleParameters` itself?

Reply:
I persued this route, but take a look at the code (will comment on it)